src/SFA.DAS.AANHub.Application/Calendars/Queries/GetCalendars/GetCalendarsQuery.cs

```diff
@@ -0,0 +1,8 @@
+﻿using MediatR;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+
+namespace SFA.DAS.AANHub.Application.Calendars.Queries.GetCalendars;
+
+public class GetCalendarsQuery : IRequest<ValidatedResponse<GetCalendarsQueryResult>>
```

Comment:
There is no  validation, do we still need validated response?

Reply:
no